README-details.md

```diff
@@ -129,6 +129,20 @@ See [Mathematics, Statistics, Probability & Probabilistic programming](./details
 
 ### Notebooks
 
+  - Apache Zeppelin notes
```

Comment:
@NavyaTatikonda Could you please add TFDIF in https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/notebooks/README.md#notebooks itself, as the Notebooks section gets linked from other places as well.

Best not to bring it into this section, it will grow and would be good to keep it in its own file.

Thanks.